### PR TITLE
add rosdep key for python-pyassimp3

### PR DESCRIPTION
**Pull request by mikeferguson**
* Used by moveit and a number of other packages - exists in python2 as "python-pyassimp"
 * Ubuntu: https://packages.ubuntu.com/focal/python3-pyassimp
 * Debian: https://packages.debian.org/unstable/python3-pyassimp
 * Fedora: https://fedora.pkgs.org/31/fedora-aarch64/python3-assimp-3.3.1-20.fc31.noarch.rpm.html

I don't understand Gentoo packaging enough to figure out what the correct key is for python3 - somebody else can hopefully add that down the road if they actually use Gentoo

### CODE REVIEW

File: rosdep/python.yaml
Comment (clalancette) [inline rosdep/python.yaml:5790]: It looks like python3-assimp is only available on debian since buster.  Since we still support stretch for Melodic, I think we need to change this to:

```
  debian:
    '*': [python3-pyassimp]
    stretch: null
```

File: rosdep/python.yaml
Comment (clalancette) [inline rosdep/python.yaml:5791]: From the link you provided, this looks like it should be:

```
fedora: [python3-assimp]
```

File: rosdep/python.yaml
Comment (clalancette) [inline rosdep/python.yaml:5792]: It looks like python3-assimp is only available on Ubuntu since bionic.  Since we still support Xenial for Kinetic, I think we need to change this to:

```
  ubuntu:
    '*': [python3-pyassimp]
    xenial: null
```

**Review by clalancette — CHANGES_REQUESTED** [review]

**Comment by mikeferguson** [comment]
@clalancette updated as per review

**Review by davetcoleman — APPROVED** [review]

**Review by clalancette — APPROVED** [review]

**Comment by davetcoleman** [comment]
@clalancette merge?

**Review by tfoote — APPROVED** [review]

**Comment by mikeferguson** [comment]
@clalancette it looks like this now has the second review from Tully - can we get this merged? It's blocking a pretty big PR on MoveIt for noetic comparability

**Pull request merged**
